apps/frontend/src/app/Formula/api.tsx

```diff
@@ -303,7 +303,7 @@ function uiDataForTeam(
     crawlObject(
       buff ?? {},
       [],
-      (x) => x.operation,
+      (x) => (x as any).operation,
```

Comment:
@lantua I am not sure how to fix this type, suggestions?

Reply:
You mean removing explicit `any`? we probably need to refactor `Data` type hierarchy or cast to `NumNode | StrNode | { operation?: undefined }`, which I don't think make things much clearer.